podcasts/Onboarding/Encourage Account Creation/Banner/InformationalBannerView.swift

```diff
@@ -0,0 +1,111 @@
+import SwiftUI
+
+struct InformationalBannerView: View {
+    @EnvironmentObject var theme: Theme
+
+    @ObservedObject var viewModel: InformationalBannerViewModel
+
+    private var backgroundColor: Color {
+        if case .profile = viewModel.bannerType {
+            if case .radioactive = theme.activeTheme {
+                return theme.primaryUi06
+            }
+            return theme.primaryUi02Active
+        }
+        switch theme.activeTheme {
+        case .indigo:
+            return theme.primaryUi02Active
+        case .contrastLight:
+            return theme.secondaryUi02
+        case .contrastDark:
+            return theme.primaryUi02Active
+        default:
+            return theme.primaryUi01
+        }
+    }
+
+    private var paddingBottom: CGFloat {
+        if case .listeningHistory = viewModel.bannerType {
+            return 0
+        }
+        return 16.0
+    }
+
+    private var paddingTop: CGFloat {
+        if case .listeningHistory = viewModel.bannerType {
+            return 22.0
+        }
+        return 16.0
+    }
+
+    var body: some View {
+        HStack(alignment: .top) {
+            Image(viewModel.bannerType.iconName)
```

Comment:
Would it make sense to add an accessibility label here? These images are showing up just with their icon names for me.

<img src="https://github.com/user-attachments/assets/7cc39ce7-5a4b-4373-9d16-e3ee0152249c" width="200" />

<img src="https://github.com/user-attachments/assets/bd641b00-d13a-4db4-8105-7489d5572920" width="200" />

<img src="https://github.com/user-attachments/assets/1703bd6a-aec2-4ebe-a7c0-c4cf78e3b52c" width="200" />